style.css

```diff
@@ -0,0 +1,5 @@
+h1 {
+  color: #000;
+
+  /* font-family: Arial, Helvetica, sans-serif; */
```

Comment:
Hi @abdulhamiid, You have done a great job with CSS :+1:
Please, consider removing comments, as it makes your code clean and more readable.